aptos-move/framework/src/natives/cryptography/ristretto255_point.rs

```diff
@@ -35,7 +36,6 @@ use std::{
     fmt::Display,
     ops::{Add, AddAssign, Mul, MulAssign, Neg, Sub, SubAssign},
 };
-
```

Comment:
nit: why not keep the line before comment

Reply:
Somehow clippy still does it ...